internal/sessiontest/legacy_test.go

```diff
@@ -68,3 +69,13 @@ func TestWithoutPairingSupport(t *testing.T) {
 
 	t.Run("StaticQRSession", apply(testStaticQRSession, nil, optionPrePairingClient))
 }
+
+func TestLinkableKeyshareResponse(t *testing.T) {
+	keyshareServer := testkeyshare.StartKeyshareServer(t, logger, irma.NewSchemeManagerIdentifier("test"))
+	defer keyshareServer.Stop()
+	client, handler := parseStorage(t, optionLinkableKeyshareResponse)
+	defer test.ClearTestStorage(t, client, handler.storage)
+	irmaServer := StartIrmaServer(t, nil)
+	defer irmaServer.Stop()
+	keyshareSessions(t, client, irmaServer)
+}
```

Comment:
This test does not work yet. This means that the new irmaclient does not support older IRMA server. This is an issue that needs to be resolved. I validated the bug by doing a session using an actual older IRMA server, so I think we can rule out a component test issue.

My guess would be that something goes wrong when the prooflist is being converted to make them work with the linkable keyshare JWTs. This happens in the function `removeKeysharePsFromProofUs`.

Reply:
Fixed in #333